infra/helper.py

```diff
@@ -124,6 +132,18 @@ def parse_args(parser, args=None):
   return parser.parse_args(args)
 
 
+def _add_external_project_args(parser):
+  parser.add_argument('--build-integration-path',
```

Comment:
I guess that works even though it seems a bit of a blunt instrument.
I would guess some projects probably would like to have their build script wherever they want but this should make our lives easier.

Reply:
It's pretty much the same as how github enforces you to have your workflows in ".github" :) Since the build script is pretty CIFuzz specific, there shouldn't be any reason they need it to live elsewhere?